nuxt.config.js

```diff
@@ -241,6 +241,7 @@ module.exports = {
     // See https://github.com/nuxt-community/axios-module#options
   },
   router: {
+    middleware: ['home-preview'],
```

Comment:
Let's move this middleware to `middleware/router/root-double-slash-redirect.js` to:
1. Organise the middleware into subdirectories from the outset
2. Name the file after its function rather than the specific circumstances motivating its introduction